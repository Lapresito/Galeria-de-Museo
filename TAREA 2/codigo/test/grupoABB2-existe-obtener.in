crearGrupoABB
crearVisitante 10 Artemio Perez 21
agregarAGrupoABB
obtenerVisitanteGrupoABB 10
crearVisitante 5 Casimira Fernandez 52
agregarAGrupoABB
crearVisitante 3 Ubaldo Fernandez 19
agregarAGrupoABB
crearVisitante 2 Esmeralda Lopez 21
agregarAGrupoABB
existeEnGrupoABB 10
existeEnGrupoABB 1
existeEnGrupoABB 2
existeEnGrupoABB 4
existeEnGrupoABB 5
existeEnGrupoABB 7
obtenerVisitanteGrupoABB 5
obtenerVisitanteGrupoABB 3
obtenerVisitanteGrupoABB 10
obtenerVisitanteGrupoABB 2
liberarGrupoABB
Fin
