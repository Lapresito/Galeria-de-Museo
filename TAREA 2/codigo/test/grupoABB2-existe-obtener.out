Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> El grupoABB ha sido creado de forma exitosa.
2> Visitante creadox de forma exitosa.
3> Se ha agregado el visitante al grupo de forma exitosa.
4> Visitante Artemio Perez
Id: 10
Edad: 21
5> Visitante creadox de forma exitosa.
6> Se ha agregado el visitante al grupo de forma exitosa.
7> Visitante creadox de forma exitosa.
8> Se ha agregado el visitante al grupo de forma exitosa.
9> Visitante creadox de forma exitosa.
10> Se ha agregado el visitante al grupo de forma exitosa.
11> El visitante con id 10 pertenece al grupo.
12> El visitante con id 1 NO pertenece al grupo.
13> El visitante con id 2 pertenece al grupo.
14> El visitante con id 4 NO pertenece al grupo.
15> El visitante con id 5 pertenece al grupo.
16> El visitante con id 7 NO pertenece al grupo.
17> Visitante Casimira Fernandez
Id: 5
Edad: 52
18> Visitante Ubaldo Fernandez
Id: 3
Edad: 19
19> Visitante Artemio Perez
Id: 10
Edad: 21
20> Visitante Esmeralda Lopez
Id: 2
Edad: 21
21> Grupo liberado con exito.
22> Fin.
